src/integration/createMetaInfo.js

```diff
@@ -35,6 +36,12 @@ function latestRepoCloner(config) {
 
     if(process.env.REGISTRY_LOCATION === 'local') return cb();
 
+    if(config.shouldUseS3){
+      logger.info('Cloning out folder from S3...');
+      latestRepoClonerS3(config, cb);
```

Comment:
We always `return` last line, not add another return, please change to `return latestRepoClonerS3(config, cb);` as well as other occurences